### PR TITLE
Create switch.ecoplug.markdown

### PR DESCRIPTION
**Pull request by adamkempenich**
Setup description for EcoPlug component in HomeAssistant.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15141
## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/switch.ecoplug.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+layout: page
+title: EcoPlug/Wion Switch
+description: "Instructions on how to set up an EcoPlug-Compatible switch within Home Assistant."
+date: 2018-06-25 14:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Switch
+ha_release: 0.71
```
Comment (frenck) [inline source/_components/switch.ecoplug.markdown]: `0.71` has been released already, maybe set this to the next major release, e.g., currently, `0.73`

File: source/_components/switch.ecoplug.markdown [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+To enable this switch, you first have to set up your devices in the EcoPlug App, and add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: ecoplug
+ ```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/switch.ecoplug.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/switch.ecoplug.markdown [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+description: "Instructions on how to set up an EcoPlug-Compatible switch within Home Assistant."
+date: 2018-06-25 14:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Switch
+ha_release: 0.71
+ha_iot_class: "Local Polling"
+---
```
Comment (frenck) [inline source/_components/switch.ecoplug.markdown]: Maybe add a logo?
That makes the documentation and the component a little more recognizable.

**Review by frenck — CHANGES_REQUESTED** [review]
@adamkempenich Thank you for this!
I've reviewed your documentation PR and left a few comments. Could you please take a look? 👍

File: source/_components/switch.ecoplug.markdown (hunk 1 above)
Comment (adamkempenich) [reply source/_components/switch.ecoplug.markdown]: Noted. Thank you! I'll make all of these changes after clearing the checklist for the device itself.

File: source/_components/switch.ecoplug.markdown [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+To enable this switch, you first have to set up your devices in the EcoPlug App, and add the following lines to your `configuration.yaml` file:
+
+{% configuration %}
+switch:
+  - platform: ecoplug
+ {% endconfiguration %}
+
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/switch.ecoplug.markdown:55]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/switch.ecoplug.markdown [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ha_category: Switch
+ha_release: 0.73
+ha_iot_class: "Local Polling"
+---
+
+This allows you to control and monitor EcoPlugs on your network. Devices are auto-discovered and added. Names are pulled from their setup names in the EcoPlugs app.
+
+To enable this switch, you first have to set up your devices in the EcoPlug App, and add the following lines to your `configuration.yaml` file:
+
+{% configuration %}
```
Comment (frenck) [inline source/_components/switch.ecoplug.markdown]: The configuration tags are for creating a list of configuration variables/options.
This is an example, which should just use markdown code...

**Review by frenck — CHANGES_REQUESTED** [review]
@adamkempenich Thank you for updating the PR.
Could you please take a look at the documentation link about configuration tags again?

In case of doubt, be sure to check out some example of other components.

File: source/_components/switch.ecoplug.markdown (hunk 5 above)
Comment (adamkempenich) [reply source/_components/switch.ecoplug.markdown]: Thanks for bearing with me on this... updating now. I think I'm figuring it out, but as always, let me know if things are still off-target!

File: source/_components/switch.ecoplug.markdown [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+ {% endconfiguration %}
+
+
+Configuration variables:
+- **scan_interval** (*Optional*): Amount of time (in seconds) before checking device's state on the network. Default is 5.
+
+
+
+
+---
```
Comment (fabaff) [inline source/_components/switch.ecoplug.markdown:19]: Needs to be the file header.

**Review by fabaff — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing this PR since the parent PR has been closed.

**Pull request closed without merging**